libflux/flux/docs.json

```diff
@@ -0,0 +1 @@
?column:string, ?groupColumns:[string]) => [B] where A: Record, B: Record"},{"pkgpath":"universe","name":"lowestCurrent","doc":"<p>lowestCurrent returns the bottom N records from all groups using the last value of each group.</p>\n","typ":"(<-tables:[A], n:int, ?column:string, ?groupColumns:[string]) => [A] where A: Record"},{"pkgpath":"universe","name":"timedMovingAverage","doc":"<p>timedMovingAverage constructs a simple moving average over windows of 'period' duration\neg: A 5 year moving average would be called as such:\nmovingAverage(1y, 5y)</p>\n","typ":"(<-tables:[A], every:duration, period:duration, ?column:string) => [B] where A: Record, B: Record"},{"pkgpath":"universe","name":"doubleEMA","doc":"<p>Double Exponential Moving Average computes the double exponential moving averages of the <code>_value</code> column.\neg: A 5 point double exponential moving average would be called as such:\nfrom(bucket: &quot;telegraf/autogen&quot;):\n|&gt; range(start: -7d)\n|&gt; doubleEMA(n: 5)</p>\n","typ":"(<-tables:[{A with _value:B}], n:int) => [C] where B: Numeric, C: Record"},{"pkgpath":"universe","name":"tripleEMA","doc":"<p>Triple Exponential Moving Average computes the triple exponential moving averages of the <code>_value</code> column.\neg: A 5 point triple exponential moving average would be called as such:\nfrom(bucket: &quot;telegraf/autogen&quot;):\n|&gt; range(start: -7d)\n|&gt; tripleEMA(n: 5)</p>\n","typ":"(<-tables:[{A with _value:B}], n:int) => [C] where B: Numeric, C: Record"},{"pkgpath":"universe","name":"truncateTimeColumn","doc":"<p>truncateTimeColumn takes in a time column t and a Duration unit and truncates each value of t to the given unit via map\nChange from _time to timeColumn once Flux Issue 1122 is resolved</p>\n","typ":"(<-tables:[{A with _time:B}], unit:duration, ?timeColumn:string) => [{A with _time:B, _time:time}] where B: Timeable"},{"pkgpath":"universe","name":"kaufmansER","doc":"<p>kaufmansER computes Kaufman's Efficiency Ratios of the <code>_value</code> column</p>\n","typ":"(<-tables:[A], n:int) => [{B with _value:float, _value:float}] where A: Record"},{"pkgpath":"universe","name":"toString","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:string}]"},{"pkgpath":"universe","name":"toInt","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:int}]"},{"pkgpath":"universe","name":"toUInt","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:uint}]"},{"pkgpath":"universe","name":"toFloat","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:float}]"},{"pkgpath":"universe","name":"toBool","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:bool}]"},{"pkgpath":"universe","name":"toTime","doc":"","typ":"(<-tables:[{A with _value:B}]) => [{A with _value:B, _value:time}]"},{"pkgpath":"universe","name":"today","doc":"<p>today returns the now() timestamp truncated to the day unit</p>\n","typ":"() => time"}]}]
```

Comment:
I don't think we should include any more generated content than is absolutely necessary. Can we just generate this on the fly and add this file to `.gitignore` so we don't commit it? Is there any value in it being in the tree?